Source/Parse/ID3TagParser.swift

```diff
@@ -8,10 +8,10 @@
 import Foundation
 
 class ID3TagParser {
+    let tagSizeParser: TagSizeParser
+    let id3TagConfiguration: ID3TagConfiguration
```

Comment:
These should be private.

Reply:
I had to make these non-private, to pass them in `Mp3FileReader.init`.

Or should I construct new instances, instead of reusing these?